README.md

```diff
@@ -8,3 +8,305 @@ To build and run this application please use the `kraft` script:
+    gcc-aarch64-linux-gnu \
+    libncurses-dev \
+    libyaml-dev \
+    flex \
+    bison \
+    git \
+    wget \
+    uuid-runtime \
+    qemu-kvm \
+    qemu-system-x86 \
+    qemu-system-arm \
+    sgabios
+```
+
+In addition, Python 2.7 needs to be installed under the name "python2" with pyYaml. This can be done with the following bash commands:
+
+    sudo apt install python2
+    sudo apt-get install python-yaml
+
+The python-yaml package can be installed in the following ways:
+
+    sudo apt install python-pip
+    pip2 install pyYaml
+
+or it can be installed with:
+
+    sudo apt-get install python-yaml
+
+## Set Up
+
+The following repositories are required for duktape:
+
+* The application repository (this repository): [`app-duktape`](https://github.com/unikraft/app-duktape)
+* The Unikraft core repository: [`unikraft`](https://github.com/unikraft/unikraft)
+* Library repositories:
+  * The duktape "library" repository: [`lib-duktape`](https://github.com/unikraft/lib-duktape)
+  * The standard C library: [`lib-musl`](https://github.com/unikraft/lib-musl)
+
+Follow the steps below for the setup:
+
+  1. First clone the [`app-duktape` repository](https://github.com/unikraft/app-duktape) in the `duktape/` directory:
+
+     ```console
+     git clone https://github.com/unikraft/app-duktape duktape
+     ```
+
+     Enter the `duktape/` directory:
+
+     ```console
+     cd duktape/
+     ```
+
+  1. While inside the `duktape/` directory, create the `.unikraft/` directory:
+
+        ```console
+        mkdir .unikraft
+        ```
+
+        Enter the `.unikraft/` directory:
+
+        ```console
+        cd .unikraft/
+        ```
+    
+  1. While inside the `.unikraft` directory, clone the [`unikraft` repository](https://github.com/unikraft/unikraft):
+
+        ```console
+        git clone https://github.com/unikraft/unikraft unikraft
+        ```
+
+  1. While inside the `.unikraft/` directory, create the `libs/` directory:
+
+     ```console
+     mkdir libs
+     ```
+
+  1. While inside the `.unikraft/` directory, clone the library repositories in the `libs/` directory:
+
+     ```console
+     git clone https://github.com/unikraft/lib-duktape libs/duktape
+
+     git clone https://github.com/unikraft/lib-musl libs/musl
+     ```
+
+  1. Get back to the application directory:
+
+     ```console
+     cd ../
+     ```
+
+     Use the `tree` command to inspect the contents of the `.unikraft/` directory.
+     It should print something like this:
+
+     ```console
+     tree -F -L 2 .unikraft/
+        .unikraft//
+        ├── libs/
+        │   ├── duktape/
+        │   └── musl/
+        └── unikraft/
+            ├── arch/
+            ├── Config.uk
+            ├── CONTRIBUTING.md
+            ├── COPYING.md
+            ├── include/
+            ├── lib/
+            ├── Makefile
+            ├── Makefile.uk
+            ├── plat/
+            ├── README.md
+            ├── support/
+            └── version.mk
```

Comment:
Provide the output of the `tree --charset=ascii -F -L 2 .unikraft/` command. It's ASCII output.